client/my-sites/media/main.jsx

```diff
@@ -1,17 +1,19 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	page = require( 'page' );
+import React from 'react';
+import page from 'page';
 
 /**
  * Internal dependencies
  */
-var MediaLibrary = require( 'my-sites/media-library' ),
-	SidebarNavigation = require( 'my-sites/sidebar-navigation' ),
-	observe = require( 'lib/mixins/data-observe' );
+import MediaLibrary from 'my-sites/media-library';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+import observe from 'lib/mixins/data-observe';
+import Dialog from 'components/dialog';
+import MediaModalDetail from 'post-editor/media-modal/detail';
 
-module.exports = React.createClass( {
+export default React.createClass( {
 	displayName: 'Media',
 
 	mixins: [ observe( 'sites' ) ],
```

Comment:
We can actually get rid of this, since this is only using selected site which is available from our Redux tree. Without the mixin, we can then port this over as a plain js class. 

Up to you if you want to leave these as TODO's for future PR's though.